src/poetry_plugin_freeze/app.py

```diff
@@ -26,8 +26,16 @@
 from poetry.core.version.markers import parse_marker, BaseMarker
 from poetry.core.constraints.version import VersionConstraint
 
+
 from poetry_plugin_export.walker import get_project_dependency_packages, walk_dependencies
 
+try:
+    from poetry_plugin_export.walker import get_project_dependency_packages2
```

Comment:
we should probably just make it unconditional and update the dependency spec for it.